internal/types/types.go

```diff
@@ -1245,6 +1265,27 @@ type ProxyComponent interface {
 	// otherwise, the `ErrorCode` of `Status` will be `Error`, and the `Reason` of `Status` will record the fail cause.
 	// error is always nil
 	AlterAlias(ctx context.Context, request *milvuspb.AlterAliasRequest) (*commonpb.Status, error)
+
+	// DescribeAlias query RootCoord to show alias for the name
+	//
+	// ctx is the context to control request deadline and cancellation
+	// req contains the request params, including collection name and new alias
+	//
+	// The `ErrorCode` of `Status` is `Success` if alter alias successfully;
+	// otherwise, the `ErrorCode` of `Status` will be `Error`, and the `Reason` of `Status` will record the fail cause.
+	// error is always nil
+	DescribeAlias(ctx context.Context, req *milvuspb.DescribeAliasRequest) (*milvuspb.DescribeAliasResponse, error)
+
+	// ListAliases query RootCoord to show all alias for the db
+	//
+	// ctx is the context to control request deadline and cancellation
+	// req contains the request params, including collection name and new alias
+	//
+	// The `ErrorCode` of `Status` is `Success` if alter alias successfully;
+	// otherwise, the `ErrorCode` of `Status` will be `Error`, and the `Reason` of `Status` will record the fail cause.
+	// error is always nil
+	ListAliases(ctx context.Context, req *milvuspb.ListAliasRequest) (*milvuspb.ListAliasesResponse, error)
```

Comment:
Alias need to be cotest with collection rename feature to make sure they can work together